enclave.go

```diff
@@ -16,74 +16,81 @@ import (
 	"fmt"
 	"log"
 	"math/big"
-	"net"
 	"net/http"
+	"net/http/httputil"
 	"net/url"
 	"os"
 	"sync"
 	"time"
 
 	"github.com/go-chi/chi/v5"
 	"github.com/go-chi/chi/v5/middleware"
-	"github.com/mdlayher/vsock"
 
 	"github.com/brave/nitriding/randseed"
-	"github.com/brave/viproxy"
 	"golang.org/x/crypto/acme/autocert"
 )
 
 const (
 	acmeCertCacheDir    = "cert-cache"
-	certificateOrg      = "Brave Software"
+	certificateOrg      = "AWS Nitro enclave application"
 	certificateValidity = time.Hour * 24 * 356
 	// parentCID determines the CID (analogous to an IP address) of the parent
 	// EC2 instance.  According to the AWS docs, it is always 3:
 	// https://docs.aws.amazon.com/enclaves/latest/user/nitro-enclave-concepts.html
 	parentCID = 3
-	// parentProxyPort determines the TCP port of the SOCKS proxy that's
-	// running on the parent EC2 instance.
-	parentProxyPort = 1080
-	pathNonce       = "/nonce"
-	pathAttestation = "/attestation"
-	pathKeys        = "/keys"
-	pathRoot        = "/"
+	// The following paths are handled by nitriding.
+	pathRoot        = "/enclave"
+	pathNonce       = "/enclave/nonce"
+	pathAttestation = "/enclave/attestation"
+	pathState       = "/enclave/state"
+	pathSync        = "/enclave/sync"
+	pathHash        = "/enclave/hash"
+	pathReady       = "/enclave/ready"
```

Comment:
I expected a subrouter pattern here, so the `/enclave` prefix is only specified once and it's easy to move all the enclave-related handlers around if necessary. But there aren't many paths and they're all in the same place, so I'm fine with this as-is.